_posts/2021-05-30-cloudnative-bot.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Using OpenFaas to run a fun twitter bot"
+description: "Using OpenFaas to run a fun twitter bot"
+date: 2021-05-20
+image: /images/2021-05-cnbot/background-twitter.png
+categories:
+ - twitter-bot
+ - use-cases
+ - functions
+author_staff_member: cpanato
+dark_background: true
+
+---
+
+Learn how Openfaas and connector-sdk can be used to run a Twitter bot
+
+## Introduction
+
+Bots are everywhere. Some are useful that can help you in your daily work, and others are collecting information about you to use to make an advertisement, and there are others to be fun and entertaining.
+
+The [@CloudNativeBot](https://twitter.com/CloudNativeBot) was created to bring some fun and also to users to use it to get some knowledge. The bot has some slash commands that can be triggered by posting a message on Twitter.
+
+The behind the technology that the bot uses will be explained in this blog post.
```

Comment:
Replace The behind the technology that the bot uses will be explained in this blog post.

With:

In this blog post I'll show you how I put the bot together and what tools I used along the way, **so that you can build your own or get ideas for your own projects with Go or OpenFaaS**

Reply:
As Dan says: "What's in it for me?" (Think of your reader.)